lib/v2/gcores/maintainer.js

```diff
@@ -2,4 +2,5 @@ module.exports = {
     '/category/:category': ['MoguCloud', 'StevenRCE0'],
     '/radios/:category?': ['eternasuno'],
     '/tag/:tag/:category?': ['StevenRCE0'],
+    '/collections/:collection': ['kudryavka1013'],
```

Comment:
```suggestion
    '/category/:category': ['MoguCloud', 'StevenRCE0'],
    '/collections/:collection': ['kudryavka1013'],
    '/radios/:category?': ['eternasuno'],
    '/tag/:tag/:category?': ['StevenRCE0'],
```
Please insert new entries in alphabetical order.